### PR TITLE
feat: add table block support

### PR DESCRIPTION
**Pull request by saltbo**
add table block support

### CODE REVIEW

File: block.go [hunk 1]
```diff
@@ -242,6 +242,27 @@ type Quote struct {
...
+type TableBlock struct {
+	BasicBlock
+	Table Table `json:"table"`
+}
+
+type Table struct {
+	TableWidth      int    `json:"table_width"`
+	HasColumnHeader bool   `json:"has_column_header"`
+	HasRowHeader    bool   `json:"has_row_header"`
+	Children        Blocks `json:"children,omitempty"`
```
Comment (jomei) [inline block.go:254]: There is no information about this field in the documentation. Do we really need `Children` here?

**Review by jomei — COMMENTED** [review]
Great job!

File: block.go (hunk 1 above)
Comment (saltbo) [reply block.go:254]: I think the documentation lose it. Because we need to save table_row by the `Children `

Comment (saltbo) [reply block.go:254]: Like the Paragraph, Callout...

**Pull request merged**